lua/Application.lua

```diff
@@ -465,6 +471,8 @@ metaTableApplication_ = {
 -- @arg data.loading An optional string with the name of loading icon. The loading available are: "balls",
 -- "box", "default", "ellipsis", "hourglass", "poi", "reload", "ring", "ringAlt", "ripple", "rolling", "spin",
 -- "squares", "triangle", "wheel" (see http://loading.io/). The default value is 'default'.
+-- @arg data.key An optional string with the Google Maps key/authentication.
+-- @arg data.template An optional table with: navbar color and title color.
```

Comment:
An optional named table with two string elements called nabber and title to describe colors for the navigation bar and for the background of the upper part of the application, respectively.